learning_resources/serializers.py

```diff
@@ -36,6 +36,40 @@ class Meta:
         fields = ["id", "name"]
 
 
+class WriteableTopicsMixin(serializers.Serializer):
+    """Class for editable topics"""
+
+    topics = WriteableSerializerMethodField()
+
+    def validate_topics(self, topics):
+        """Validator for topics"""  # noqa: D401
```

Comment:
IMO: here and elsewhere, might as well just write it the way the linter suggests.

```suggestion
        """Validate specified topics exist.""
```